Editor/AGS.Editor/Utils/Validation.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace AGS.Editor.Utils
+{
+    public class Validation
+    {
+        public static bool FilenameIsValid(string filename)
+        {
+            return !(filename.IndexOfAny(Path.GetInvalidFileNameChars()) >= 0);
+        }
+
+        public static bool PathDoesNotExist(string path)
+        {
+            if (File.Exists(path))
+                return false;
+
+            if (Directory.Exists(path))
+                return false;
+
+            return true;
+        }
+
+        public static bool PathIsAbsoluteDriveLetter(string path)
+        {
+            return Regex.IsMatch(path, @"^[a-zA-Z]:\\");
+        }
+
+        public static bool PathIsAbsolute(string path)
+        {
+            bool rooted = false;
+
+            try
+            {
+                if (Path.IsPathRooted(path))
```

Comment:
It's a small thing, but expressing this in one line just is easier on the eyes `rooted = Path.IsPathRooted(path)`

Reply:
Do you mean instead of using if and `rooted = true`? Logically I am only interested in setting it to true, so to me the reading is clearer when the block logic resembles how I would explain it alongside the chance of an exception being thrown, but I am just me so I'll change it if you think the one line is better.